pkg/controller/controller.go

```diff
@@ -20,26 +20,28 @@
 package controller
 
 import (
+	"context"
 	"fmt"
-	"os"
-	"os/signal"
-	"path/filepath"
 	"reflect"
 	"strings"
-	"syscall"
 	"time"
 
+	"k8s.io/apimachinery/pkg/api/meta"
+	"k8s.io/client-go/dynamic"
+	"k8s.io/client-go/dynamic/dynamicinformer"
+
```

Comment:
```suggestion
```

Reply:
Let's put 3rd party and local packages into separate groups

Reply:
I used the following command it went through all files, but didn't change anything:

```
for file in $(find . -name '*.go'); do
    goimports-reviser -file-path $file -rm-unused -set-alias -format -local github.com/infracloudio/botkube -project-name github.com/infracloudio/botkube -list-diff
done
```

I didn't do any newlines in imports manually. I disabled "Optimize imports on the fly" in IntellIJ, maybe that'll help.
I will fix the imports manually for now.